app/Post_Types/Archive_Item/Post_Meta.php

```diff
@@ -50,37 +73,108 @@ public function add_post_meta() {
 			->add_fields( array(
 				$this->get_date_field(),
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
 
 
 		$this->container = Container::make( 'post_meta', 'Related Items' )
 			->where( 'post_type', '=', Archive_Item::NAME )
 			->add_fields( array(
 				$this->get_field_related(),
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
 
 		$this->container = Container::make( 'post_meta', 'Audio' )
 			->where( 'post_type', '=', Archive_Item::NAME )
 			->add_fields( array(
 				$this->get_field_audio(),
 				$this->get_field_audio_oembed()
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
 
 		$this->container = Container::make( 'post_meta', 'Video' )
 			->where( 'post_type', '=', Archive_Item::NAME )
 			->add_fields( array(
 				$this->get_field_video_oembed()
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
 
 		$this->container = Container::make( 'post_meta', 'Document' )
 			->where( 'post_type', '=', Archive_Item::NAME )
 			->add_fields( array(
 				$this->get_field_document()
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
+
+	}
+
+	/**
+	 * Adds one or more classes to the body tag in the dashboard.
+	 *
+	 * @param  \WP_Post $post Current field post.
+	 * @return String          Class name.
+	 */
+	public function get_class( $post ) {
+		$type = carbon_get_the_post_meta( FieldPostMeta::FIELD_TYPE, 'carbon_fields_container_field_variables' );
+		$name = null;
+		switch ($type) {
+			case Text_Field::NAME:
+				$name = Text_Field::class;
+				break;
+			case Date_Field::NAME:
+				$name = Date_Field::class;
+				break;
+			case CPT_Field::NAME:
+				$name = CPT_Field::class;
+				break;
+		}
+		return $name;
+	}
+
+	public function get_field($type) {
```

Comment:
https://github.com/NCPR/diviner-wp-archive-theme/pull/2

Reply:
I still have to catch up about the specifics but that seems like a good approach.